### DISCUSSION TITLE
Tracking usage of a particular discount code

### DISCUSSION THREAD

**Discussion by TimB111** [liedekef/events-made-easy#91] [General]
Is there a way to see all of the bookings for a particular discount code? I can filter the discounts list to see the discount but can't see where/when it was used. 

I'd also like to be able to give a regular event guest access to a page where they can see the usage of their discount code (event name, event date, discount amount, remaining discount). They will share their discount code with friends so I can't try to track it by email address.

**Comment by liedekef** [comment]
In the bookings overview, you have "Used discount codes" as a column.
A guest can't check anything though ... I know this is linked to the other question you asked, but currently the only way a guest knows what EME has as info on it, is via the [eme_gdpr] shortcode (and unfortunately - for now - that doesn't include the bookings (past nor future).

**Reply by TimB111** [reply]
Thanks, yes I can see the "Used discount codes" column but can't sort or filter by this column so need to manually check through pages of bookings to find all of them.

**Reply by liedekef** [reply]
Did you try exporting the bookings to CSV (either per event, or using the CSV-export button) and then search in excel? While it is a workaround, it does actually help for more complicated things to check :-)

**Reply by TimB111** [reply]
This works, thank you!
